sysbuild/Kconfig.secureboot

```diff
@@ -30,8 +30,8 @@ config SECURE_BOOT_NETWORK_BOARD_TARGET_CPUCLUSTER
 	default "cpunet" if SOC_NRF5340_CPUAPP
 	depends on SECURE_BOOT_NETCORE
 	help
-	  Remote board when building this sample.
-	  If not set, the board given to sysbuild is used.
+	  Remote board when building the secure boot application for the network core. If not set,
+	  the board given to sysbuild is used.
```

Comment:
text says remote board, but default value and setting name indicates that only the CPU cluster should be given.